core/src/main/java/io/grpc/internal/TransportSet.java

```diff
@@ -222,30 +232,32 @@ private boolean isAttachedToActiveTransport() {
     @Override
     public void transportReady() {
       synchronized (lock) {
-        log.info("Transport '" + transport + " is ready");
+        log.info("Transport '" + transport + "' for " + server + " is ready");
```

Comment:
Converted to the long form. I would like them to be info. These connection life-cycle events are usually checked when investigating a production issue. We should have them on by default.